app/gui/src/main/scala/org/bitcoins/gui/ContractGUI.scala

```diff
@@ -0,0 +1,274 @@
+package org.bitcoins.gui
+
+//import akka.actor.ActorSystem
+import org.bitcoins.cli.CliCommand.{
+  AcceptDLCCliCommand,
+  AddDLCSigsAndBroadcastCliCommand,
+  CreateDLCOffer,
+  SignDLCCliCommand
+}
+import org.bitcoins.core.protocol.dlc.models.DLCStatus
+import org.bitcoins.core.protocol.tlv.{
+  ContractInfoV0TLV,
+  DLCAcceptTLV,
+  DLCOfferTLV,
+  DLCSignTLV,
+  LnMessageFactory,
+  OracleAnnouncementV0TLV
+}
+import org.bitcoins.gui.contract.GlobalContractData
+import org.bitcoins.gui.dlc.DLCPaneModel
+import org.bitcoins.gui.dlc.dialog.{
+  AcceptOfferDialog,
+  BroadcastDLCDialog,
+  CreateDLCOfferDialog,
+  DLCDialogContainer,
+  SignDLCDialog,
+  ViewDLCDialog
+}
+import org.bitcoins.gui.util.GUIUtil
+import scalafx.beans.property.StringProperty
+import scalafx.geometry._
+import scalafx.scene.Parent
+import scalafx.scene.control.{
+  ContextMenu,
+  MenuItem,
+  TableColumn,
+  TableView,
+  TextField
+}
+import scalafx.scene.layout._
+
+import java.io.File
+import java.nio.file.Files
+import java.text.SimpleDateFormat
+import java.util.Date
+import scala.util.{Failure, Success}
+
+class ContractGUI(glassPane: VBox) {
+
+  def fetchStartingData(): Unit = {
+    // Nothing to fetch for state yet...
+  }
```

Comment:
can delete?

Reply:
Yeah, I had this in to match pattern assuming it would probably see some use later. I can remove it.